test/unit/helpers.py

```diff
@@ -0,0 +1,80 @@
+"""Unit tests for pudl.transform.eia module."""
```

Comment:
I think this should be `Unit tests for pudl.helpers` module, but I love this!

Reply:
Ah yeah sorry. Cut and paste error from my previous attempt. Also, I need to rename this module `helpers_test.py` or it won't actually be detected by pytest. Duh.